### ISSUE TITLE
Test out 0.9.0rc ACA-Py update

### ISSUE THREAD

**Issue by loneil** [bcgov/traction#730]
I tried upgrading Traction to 0.9.0 when the release was made on Friday afternoon. Getting abandoned credentials on the attempts to issue once the holder accepts. This error is new and different from the one when we tried the upgrade to askar-only before that was fixed with `0.8.3pre` (which is in Dev Traction right now).
Though still related to Tails files.

See attached PR for details. Haven't had luck tracing any error yet other than the error_msg result shown in the PR comments

Creating ticket so @esune , @hiteshgh , @swcurran can prioritize if someone else should look at as I am away out of service monday/tuesday/wednesday this week.

**Comment by swcurran** [comment]
Absolute top priority, folks — we really need to understand this one. Sleep well tonight so we’re all ready on Monday.  :-)

**Comment by jleach** [comment]
I think this impacts proof requests also. In AFJ I'm seeing v1 proofs `https://didcomm.org/present-proof/1.0/request-presentation` get to the "abandoned" state which is not part of the v1 protocol.

```
{
  "_tags": {
    "connectionId": "53d160c3-41cf-4fd7-8dc8-de21527d6e12",
    "state": "abandoned",
    "threadId": "439d70f8-6f52-435b-8dd8-04bd5efe2517"
  },
  "metadata": {},
  "id": "44568600-6420-4c6d-b79d-24a347a0b3d4",
  "createdAt": "2023-07-18T21:15:01.227Z",
  "protocolVersion": "v1",
  "state": "abandoned",
  "connectionId": "53d160c3-41cf-4fd7-8dc8-de21527d6e12",
  "threadId": "439d70f8-6f52-435b-8dd8-04bd5efe2517",
  "updatedAt": "2023-07-18T21:15:11.276Z",
  "errorMessage": "abandoned: abandoned"
}
```

**Comment by esune** [comment]
@jleach the issue was a plugin mismatch in Traction (see #279 for further details). The agent deployment the BC Wallet showcase is depending on will be updated soon after the PR is merged.

**Comment by esune** [comment]
Resolved by #729
